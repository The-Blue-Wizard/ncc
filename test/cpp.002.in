/* allow identical redefinitions */

#define X 2 3 ( ) "hello"
#define X 2 3 ( ) "hello"
X
