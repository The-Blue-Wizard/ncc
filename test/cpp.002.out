# 1 "cpp.002.in"




2 3 ( ) "hello" 
